### PR TITLE
Created the vertical_scaling.md file

### LINKED ISSUE #157: Vertical scaling definition
Please follow format defined in[ style guide](https://github.com/cncf/glossary/blob/main/style-guide.md).

### PR DESCRIPTION
**Pull request by reapedjuggler**
Created the `vertical_scaling.md` file for vertical scaling definition it resolves #157

### CODE REVIEW

**Comment by reapedjuggler** [comment]
Let me know if there are any more changes :)

File: definitions/vertical_scaling.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+title: Vertical Scaling
+status: Completed
+category: Concept
+---
+
+## Vertical Scaling
+
+### What it is
+
+The ability for an application to scale, typically by adding CPU and memory as the workload increases. Also known as “scale up”, this refers to adding resources to a single node.
```
Comment (CathPag) [inline definitions/vertical_scaling.md:11]: @JasonMorgan, I think we should mirror the horizontal scaling definition a little. Here's my suggestion:

Vertical scaling, also known as "scaling up and down", is a technique where a system's capacity is increased by adding CPU and memory to individual [nodes](https://github.com/cncf/glossary/blob/main/definitions/nodes.md) as the workload increases. Let's say, you have a computer of 4GB RAM and want to increase its capacity to 16GB RAM, scaling it vertically means switching to a 16GB RAM system. (Please refer to [horizontal scaling](https://github.com/cncf/glossary/blob/main/definitions/horizontal_scaling.md) for a different scaling approach.)

File: definitions/vertical_scaling.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+### Problem it Addresses
+
+[Scalability](https://github.com/cncf/glossary/blob/main/definitions/scalability.md) is essential in that it contributes to competitiveness, efficiency, reputation and quality. Vertical scaling is on one of the two kinds Vertical, [Horizontal](https://github.com/cncf/glossary/blob/main/definitions/horizontal_scaling.md) of scalability.
+
+As demand for an application grows beyond the current capacity of that application instance, we need to find a way to scale (add capacity to) the system. We can either add more nodes to the system ([horizontal scaling](https://github.com/cncf/glossary/blob/main/definitions/horizontal_scaling.md)) or more compute resources to existing nodes (vertical scaling).
+
+### How it helps
```
Comment (CathPag) [inline definitions/vertical_scaling.md:18]: This isn't talking about the problem it addresses but rather what it is. Here's my suggestion:

Vertical scaling allows an app to process more requests and do more work concurrently without adding more nodes to the system (scale horizontally).

**Review by CathPag — COMMENTED** [review]
@JasonMorgan for our discussion in an hour

File: definitions/vertical_scaling.md (hunk 2 above)
Comment (reapedjuggler) [reply definitions/vertical_scaling.md:18]: Thank you for proposing the changes, shall i raise a new commit regarding the proposed changes ?

Comment (CathPag) [reply definitions/vertical_scaling.md:18]: Not yet. Discussed with Jason and your definition works but needs more context. Below is an updated version (a mix of both versions). Let me review with Jason on Friday. Hoping to get this approved and merged before the weekend. 

Vertical scaling allows you to resize your server without changing the application code. That contrasts to horizontal scaling, where the app must be replicable to scale, potentially requiring code updates. Vertical scaling increases the capacity of an existing application by adding compute resources, allowing the app to process more requests and do more work concurrently.

**Comment by reapedjuggler** [comment]
@JasonMorgan is it okay now ?

**Comment by JasonMorgan** [comment]
Yes it is! Great work!

**Comment by reapedjuggler** [comment]
Thank you :smile:

**Pull request merged**